wrappers/python/tests/model/test_gpt2_mlp.py

```diff
@@ -0,0 +1,184 @@
+        vocab_embed_dim_tile=params.hidden_size,
+        hidden_size=params.hidden_size,
+        hidden_size_tile=params.hidden_size_tile,
+        intermediate_size=params.intermediate_size,
+        intermediate_size_tile=params.intermediate_size_tile,
+        n_head=torch_layer_config.n_head,
+        n_head_tile=torch_layer_config.n_head,
+        dtype=dtype
+    )
+    x_shape = [params.hidden_size, params.seq_len, params.n_batch]
+    x_basetile = [params.hidden_size_tile,
+                  params.seq_len_tile,
+                  params.n_batch_tile]
+    x_traits = TensorTraits(x_shape, x_basetile)
+    x_distr = [0] * x_traits.grid.nelems
+    x_type = dtype2nntile[dtype]
+    x_value = x_type(x_traits, x_distr, 0)
+    x_grad = x_type(x_traits, x_distr, 0)
+    X = TensorMoments(x_value, x_grad, grad_required=True)
+    gen = np.random.default_rng(42)
+    x_random = gen.standard_normal(x_shape, dtype=np.float32)
+    x_nntile = np.array(x_random, dtype=np.float32, order="F")
+    x_value.from_array(x_nntile)
+    x_torch = torch.Tensor(x_nntile.T)
+    nntile_layer, _ = GPT2MLP_nntile.from_torch(torch_layer, X,
+                                                nntile_config, 0)
+    nntile_layer.clear_gradients()
+    y_grad_random = gen.standard_normal(x_shape, dtype=np.float32)
+    y_grad_nntile = np.array(y_grad_random, dtype=np.float32, order="F")
+    nntile_layer.activations[-1].grad.from_array(y_grad_nntile)
+    y_grad_torch = torch.Tensor(y_grad_nntile.T)
+    return torch_layer, nntile_layer, x_torch, y_grad_torch
+
+
+@pytest.mark.parametrize('params', [
+    pytest.param(single_tile, id='single_tile'),
+    pytest.param(multiple_tiles, id='multiple_tiles'),
+])
+@pytest.mark.parametrize('dtype', [
+    'fp32',
+    pytest.param('fp32_fast_tf32', marks=nocuda),
+    pytest.param('bf16', marks=nocuda),
+])
+class TestGPT2MLP:
+
+    def test_coercion(self, starpu_simple, torch_rng,
+                      params: GPT2MLPTestParams, dtype: str):
+        torch_layer, nntile_layer, _, _ = generate_inputs(params, dtype)
+        torch_layer_other = nntile_layer.to_torch()
+        nntile_layer.unregister()
+
+        rtol = dtype2tol[dtype]['rtol']
+        for (n1, p1), (n2, p2) in zip(torch_layer.named_parameters(),
+                torch_layer_other.named_parameters()):
+            assert n1 == n2
+            assert torch.norm(p1 - p2) <= rtol * torch.norm(p1)
+
+    def test_forward(self, starpu_simple, torch_rng,
+                     params: GPT2MLPTestParams,
+                     dtype: str):
+        torch_layer, nntile_layer, x, _ = generate_inputs(params, dtype)
+        y = torch_layer(x)
+        nntile_layer.forward_async()
+        y_nntile = torch.Tensor(to_numpy(nntile_layer.activations[-1].value).T)
+        nntile_layer.unregister()
+        rtol = dtype2tol[dtype]['rtol']
+        assert torch.norm(y - y_nntile) <= rtol * torch.norm(y)
+
+    def test_forward_backward(self, starpu_simple, torch_rng,
```

Comment:
rename to `test_backward` and swap check of output by check of gradient over input